plugins/connections/ui/src/main/java/org/pentaho/di/connections/ui/tree/ConnectionFolderProvider.java

```diff
@@ -71,4 +111,59 @@ public void create( AbstractMeta meta, TreeNode parent ) {
     refresh( meta, createTreeNode( parent, getTitle(), getTreeImage() ), null );
 
   }
+
+  // this could become common for all the types.
+  enum LEVEL {
+    PROJECT( "Project" ),
```

Comment:
Not sure about letting the project concept in here..
Perhaps an identifier could come from the bowl for project and default?

Reply:
coming from the bowl would only work for Global and Project... wouldn't work for Local. 
Also, I'm assuming we'll need to localize these. hrm. I should probably have done that already. I'll open a subtask for that.